cultivar_app.py

```
@@ -198,25 +274,107 @@ def rate_limit_exceeded(e):
+                nonce = getattr(request, 'csp_nonce', None)
+
+            # try to read from the talisman extension if present
+            if not nonce:
+                tal = app.extensions.get('talisman') if hasattr(app, 'extensions') else None
+                if tal:
+                    # Some versions store a callable or attribute for nonce
+                    candidate = getattr(tal, 'csp_nonce', None) or getattr(tal, '_csp_nonce', None)
+                    if callable(candidate):
+                        try:
+                            nonce = candidate()
+                        except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)